bedrock/bedrock-v407/src/main/java/com/nukkitx/protocol/bedrock/v407/serializer/StartGameSerializer_v407.java

```diff
@@ -71,6 +81,10 @@ public void serialize(ByteBuf buffer, BedrockPacketHelper helper, StartGamePacke
         buffer.writeIntLE(packet.getUnknownInt0());
         buffer.writeIntLE(packet.getUnknownInt1());
 
+        // @Todo
+        buffer.writeByte(0);
+        buffer.writeByte(0);
```

Comment:
bool - nether type
bool - force experimental gameplay

Reply:
nether type is a bool?

Reply:
Probably means old vs new nether